Dockerfile.template

```diff
@@ -36,24 +36,10 @@ RUN apk add libstdc++ unixodbc unixodbc-dev && \
     pecl install pdo_sqlsrv && docker-php-ext-enable pdo_sqlsrv
 
 
-# install Oracle Instant client
-RUN git clone https://github.com/adrianharabula/php7-with-oci8.git && (cd php7-with-oci8 && git reset --hard d57bf99af398c0cd0e1c2a4c9f8006590600c951) && \
-    mv php7-with-oci8/instantclient/12.2.0.1.0/* /tmp && rm -rf php7-with-oci8
-RUN unzip /tmp/instantclient-basiclite-linux.x64-12.2.0.1.0.zip -d /usr/local && \
-    unzip /tmp/instantclient-sdk-linux.x64-12.2.0.1.0.zip -d /usr/local && \
-    unzip /tmp/instantclient-sqlplus-linux.x64-12.2.0.1.0.zip -d /usr/local && \
-    ln -s /usr/local/instantclient_12_2 /usr/local/instantclient && \
-    ln -s /usr/local/instantclient/libclntsh.so.12.1 /usr/local/instantclient/libclntsh.so && \
-    ln -s /usr/local/instantclient/sqlplus /usr/bin/sqlplus
-ENV LD_LIBRARY_PATH /usr/local/instantclient:$LD_LIBRARY_PATH
-
-# install pdo_oci PHP extension
-RUN apk add libaio libnsl libc6-compat && \
-    docker-php-ext-configure pdo_oci --with-pdo-oci=instantclient,/usr/local/instantclient && \
-    ln -s /usr/lib/libnsl.so.2 /usr/lib/libnsl.so.1 && \
-    ln -s /lib/libc.so.6 /usr/lib/libresolv.so.2 && \
-    ln -s /lib64/ld-linux-x86-64.so.2 /usr/lib/ld-linux-x86-64.so.2 && \
-    docker-php-ext-install pdo_oci
+# install Oracle Instant client & pdo_oci PHP extension
+COPY --from=mlocati/php-extension-installer /usr/bin/install-php-extensions /usr/local/bin/
+RUN install-php-extensions oci8
+RUN docker-php-ext-install pdo_oci
```

Comment:
@mlocati can support for `pdo_oci` be easily added?

Reply:
@mvorisek Sure! See https://github.com/mlocati/docker-php-extension-installer/pull/234